### PR TITLE
Implemented local storage on contact form

### PR DESCRIPTION
**Pull request by gracekabaghe**
- When the user changes the content of any input field, the data is saved to the local storage.
- When the user loads the page if there is any data in the local storage the input fields are pre-filled with this data.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,10 +23,11 @@
 </script>
 </head>
 <body>
```
Comment (SajjadAhmad14) [inline index.html:25]: The whole web page is blur so I was unable to check the project against the requirements
blur
.

File: local.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const contactForm = document.querySelector('form');
+
+contactForm.addEventListener('keyup', (e) => {
```
Comment (Mihndim2020) [inline local.js:3]: - [ ] The contact form information  persist on page reloads, after filling the form, and not submitting it, the entered values should be saved in the browser, when visiting the page the form inputs should be contain, be auto populated with, the values entered in the previous visit to the page, If at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty. Please kindly fix that as this is a requirement for this project. Please use `change` event listener on the form input fields. Checkout this [article](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event)

Comment (gracekabaghe) [reply local.js:3]: Hi @Mihndim2020 - just want to confirm if the form is not submitting: I noticed someone submitted this https://share.getcloudapp.com/E0uK7AgX - is that you. Thanks for the article. However, I need a bit of clarification on the issue. Here is how it works on my end https://share.getcloudapp.com/E0uK7AOv -

**Review by dasileker — APPROVED** [review]
### **Hi @gracekabaghe,** 


### **Optional**

- The `window.onload => nameValue` is causing an issue, you can simply do `fromData to retrieve the value input of each.


### **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 


Approved



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**